controls/cf_serverd.cf

```diff
@@ -141,6 +141,12 @@ bundle server mpf_default_access_rules()
           resource_type => "query",
           admit_ips => { @(def.mpf_access_rules_collect_calls_admit_ips) };
 
+      "$(sys.workdir)/cmdb/$(connection.key)/" -> { "ENT-6788" }
```

Comment:
Indentation seems off, this is inside the enterprise and policy_server class guard.

Reply:
Isn't that the only type of host that should be promising access to the data?

Reply:
Oh indentation

Reply:
The indentation was wrong for the promise above.

Reply:
Fixed.